Azure/azure-cli
# [Storage] `az storage account network-security-perimeter`: New command group to support network security perimeter associations management

**Related command**

`az storage account network-security-perimeter list/show/reconcile`

**Description**

Private preview support for https://github.com/Azure/azure-rest-api-specs/pull/21557

**Testing Guide**

- Create a NSP using portal (preview)
- Associate a storage account (locates in eastus2euap) with NSP
- Try `az storage account network-security-perimeter list/show/reconcile`


---

This checklist is used to make sure that common guidelines for a pull request are followed.
